server/mailet/integration-testing/src/test/java/org/apache/james/transport/mailets/ICSAttachmentWorkflowTest.java

```diff
@@ -550,6 +551,7 @@ public void setup() throws Exception {
     @After
     public void tearDown() throws Exception {
         jamesServer.shutdown();
+        amqpRule.readAll();
```

Comment:
is this fixing the big stack trace we get on every test?

Reply:
No, my change is focused on testing speed. Not on possible stacktrace in the output.